src/Bladeburner/IBladeburner.ts

```diff
@@ -70,13 +72,8 @@ export interface IBladeburner {
   getGeneralActionNamesNetscriptFn(): string[];
   getSkillNamesNetscriptFn(): string[];
   startActionNetscriptFn(player: IPlayer, type: string, name: string, workerScript: WorkerScript): boolean;
-  getActionTimeNetscriptFn(player: IPlayer, type: string, name: string, workerScript: WorkerScript): number;
-  getActionEstimatedSuccessChanceNetscriptFn(
-    player: IPlayer,
-    type: string,
-    name: string,
-    workerScript: WorkerScript,
-  ): [number, number];
+  getActionTimeNetscriptFn(person: IPerson, type: string, name: string): number | string;
```

Comment:
Why did you change the return type? You specifically check that the returned value is not a `string` from what I see. Same for `getActionEstimatedSuccessChanceNetscriptFn`

Reply:
Changing the return type allows for the removal of the dependency on `WorkerScript` which was used for logging in the event of errors. This way the function is more functional allowing the caller to handle any side effects. This allows for sleeves and player to handle the errors in what is most appropriate for their respective situation.